src/components/Authentication/Login.js

```diff
@@ -37,7 +37,7 @@ function Login(props) {
                 <title>UBC BizTech - Log In or Sign Up</title>
             </Helmet>
             <h1> Login </h1>
-            <Button onClick={() => Auth.federatedSignIn({ provider: 'Google' }).then(() => Auth.currentAuthenticatedUser({ bypassCache: true }))} variant="contained" color="primary">
+            <Button onClick={() => Auth.federatedSignIn({ provider: 'Google' })} variant="contained" color="primary">
```

Comment:
Why was this removed? (tbh can't remember what it's for)

Reply:
Honestly not sure either.. I removed this because it didn't really change anything. `Auth. currentAuthenticatedUser` is already called on every render of `route.js`, so I didn't see why this was needed here.

Reply:
:seemsgood: